packages/core/src/contracts/tcr/civilTCR.ts

```diff
@@ -309,7 +309,7 @@ export class CivilTCR extends BaseWrapper<CivilTCRContract> {
       ._ChallengeStream({}, { fromBlock })
       .map(e => new Listing(this.ethApi, this.instance, e.args.listingAddress))
       .concatMap(async l => l.getListingWrapper())
-      .concatFilter(l => isListingInAppealChallengeCommitPhase(l.data));
+      .concatFilter(l => isInAppealChallengeCommitPhase(l.data));
```

Comment:
i think these concat filters are bad because they obscure the number of events actually being fired.

Reply:
well let's talk about that later, not a part of this PR